espnet/nets/pytorch_backend/lm/default.py

```diff
@@ -27,6 +27,8 @@ def add_arguments(parser):
                             help='Number of hidden layers')
         parser.add_argument('--unit', '-u', type=int, default=650,
                             help='Number of hidden units')
+        parser.add_argument('--embed-unit', type=int, default=128,
```

Comment:
Currently, the default uses `args.unit`, so could you also set it like that instead of `default=128`? (maybe we can just set with `default=None`?

Reply:
Actually, I think about it last night. I agree with change default with 650 keeping same with unit.

Reply:
I think it's better to make `default=None`.
Then, it just uses the same dimension as the number of hidden units according to https://github.com/espnet/espnet/blob/860cbd249f5d2ed48bd3f97cd82b984d7a7939a3/espnet/nets/pytorch_backend/lm/default.py#L242
Thus, this becomes the exact same behavior unless your new option of `--embed-unit` is specified.
If you agree, please change it, and also add such comments in the argument help.

Reply:
ok, I can do it, but which place you suggest my add some comment? in parser.add_argument 's help parameter?

Reply:
yes!